File: programmers/python/level1/sort_int_place_value_desc.py
def solution(n):
    a = sorted(map(int, str(n)))
    return sum( a[i-1] * 10 ** (i-1) for i in range(1, len(a)+1))

def solution(n) :
    ls = list(str(n))
    ls.sort(reverse = True)
    return int("".join(s))

print(solution(118372))

File: programmers/java/src/level1/SortIntPlaceValueDesc.java
package level1;

import java.util.Arrays;
import java.util.Collections;

public class SortIntPlaceValueDesc {
    public static void main(String[] args) {
        System.out.println(solution(8000000000L));
    }

    public static long solution(long n) {
        String[] split = ("" + n).split("");
        Arrays.sort(split, Collections.reverseOrder());
        return (long) Long.parseLong(String.join("", split));
    }
}
